app.js

```diff
@@ -0,0 +1,93 @@
+const form = document.querySelector('#new-task-form');
+const button = document.querySelector('#button');
+const list =  document.querySelector('#task-list');
+const field = form['new-task-input'];
+
+
+const todoList = [];
+
+createEditButton = () => {
+    const editButton = document.createElement('button');
+    editButton.classList.add = 'edit';
+    editButton.innerText = 'Edit';
+
+    //edit task handler
+    editButton.addEventListener('click', (e) => {
+        //logic
+    })
+
+    return editButton;
+}
+
+createDeleteButton = () => {
+    const deleteButton = document.createElement('button');
+    deleteButton.classList.add = 'delete';
+    deleteButton.innerText = 'Delete';
+
+    //delete task handler
+    deleteButton.addEventListener('click', (e) => {
+        //logic
+    })
+
+    return deleteButton;
+}
+
+
+createTask = (localTodo) => {
+
+    const divEl = document.createElement('div');
+    divEl.classList.add = 'div-to-add';
+
+    divEl.classList.add = "todoTasks"
+
+    const inputEl = document.createElement('input');
+    inputEl.classList.add("text");
+    inputEl.type = "text";
+    inputEl.value = localTodo.text;
+    inputEl.setAttribute("readonly", "readonly")
+    divEl.appendChild(inputEl);
+
+
+    //Creating edit button for the task
+    const editButton = createEditButton();
+    divEl.appendChild(editButton);
+
+    //Creating delete button for the task
+    const deleteButton = createDeleteButton();
+    divEl.appendChild(deleteButton);
+
+    return divEl;
+
+}
```

Comment:
Multiple issues
1. The function is doing too much. Reduce it to follow Single Responsibility pattern. A function should do one unit of task
2. Don't create global variables or using `var`. `const` or `let` is always preferred.
3. `Element.classList.add` is a function not a property. Always refer to mdn to know about DOM api.
```suggestion
const createTodoElement = (todo) => {

    const todo$ = document.createElement('div');
    todo$.classList.add('todo')

    return todo$;
}
```